{"name": "test_zx1", "status": "broken", "statusDetails": {"message": "AttributeError: 'Login' object has no attribute 'gp_login'", "trace": "driver_fixture = <appium.webdriver.webdriver.WebDriver (session=\"0e7b83d5-90bd-4243-ba0f-6351e0c81c1e\")>\n\n    @pytest.fixture(scope='session',autouse=False)\n    def gp_login_fixture(driver_fixture):\n        csv_file=BASE_DIR + '\\datas\\login.csv'\n        print(\"csv_file:%s\"%csv_file)\n        driver=driver_fixture\n        com=Common(driver)\n        data=com.get_csv_data(csv_file,2)\n        time.sleep(2)\n        trd=First(driver)\n        trd.to_login()\n        logn=Login(driver)\n>       logn.gp_login(data[0],data[1])\nE       AttributeError: 'Login' object has no attribute 'gp_login'\n\nconftest.py:54: AttributeError"}, "start": 1652795334072, "stop": 1652795334072, "uuid": "c039eb6d-fcc1-40f4-af5c-778c482e1151", "historyId": "df32e6c380106ba5ba8bf8fe03f2a486", "testCaseId": "cf47af9319c62ba664346435549117f5", "fullName": "testGp.test_new_Gp.TestGp#test_zx1", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('back_fixture')"}, {"name": "parentSuite", "value": "testGp"}, {"name": "suite", "value": "test_new_Gp"}, {"name": "subSuite", "value": "TestGp"}, {"name": "host", "value": "Win102021AVUCSJ"}, {"name": "thread", "value": "10404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testGp.test_new_Gp"}]}